cluster/gce/container-linux/configure-helper.sh

```diff
@@ -341,7 +341,9 @@ EOF
   fi
 }
 
-function create-kubelet-kubeconfig {
+# Arg 1: the address of the API server
+function create-kubelet-kubeconfig() {
+  local apiserver_address="${1}"
```

Comment:
want to make sure we error if called without an address... no telling what is making use of `create-kubelet-kubeconfig`